tox.ini

```diff
@@ -1,11 +1,16 @@
 [tox]
 downloadcache = {toxworkdir}/_download/
-envlist = {py27,py34}-django{16,17}
+envlist = py{27,34}-django{16,17,18}-cms{30,31,32}
 
 [testenv]
-commands = {envbindir}/python examples/manage.py test bs3demo --settings=bs3demo.settings
+commands = {envbindir}/python test_settings.py
 deps =
-    django16: django>=1.6,<1.7
-    django17: django>=1.7,<1.8
-    py{26,27}: wsgiref
+    django16: django<1.7
+    django17: django<1.8
+    django18: django<1.9
+    cms30: https://github.com/divio/django-cms/archive/support/3.0.x.zip
+    cms31: https://github.com/divio/django-cms/archive/support/3.1.x.zip
+    cms32: https://github.com/divio/django-cms/archive/develop.zip
+    py27: wsgiref
     -rexamples/bs3demo/requirements.txt
```

Comment:
TO DO: Divinding requirements into basic package requirements and specific bs3demo project requirements.